cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/mod.rs

```diff
@@ -18,9 +18,10 @@ use crate::imports::*;
 mod asset_transfers;
 mod claim_assets;
 mod send_xcm;
-mod snowbridge;
 mod teleport;
 
+mod snowbridge;
+
```

Comment:
Probably an unnecessary change.

Reply:
We do need that there is some common code [here](https://github.com/Snowfork/polkadot-sdk/blob/b19c1e11e382e649de38afc46bb93dd9eb2d5a1f/cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/mod.rs#L67-L68).